ui/packages/tidb-dashboard-for-clinic-cloud/src/apps/SlowQuery/context.ts

```diff
@@ -80,6 +80,30 @@ class DataSource implements ISlowQueryDataSource {
   slowQueryDownloadTokenPost(request: any, options?: ReqConfig) {
     return client.getInstance().slowQueryDownloadTokenPost({ request }, options)
   }
+
+  slowQueryAnalyze(start: number, end: number) {
+    return client
+      .getAxiosInstance()
+      .get(`/slow_query/analyze?begin_time=${start}&end_time=${end}`)
+  }
+
+  promqlQuery(query: string, time: number, timeout: string) {
+    return client
+      .getAxiosInstance()
+      .get(
+        `/slow_query/vm_query?query=${query}&time=${time}&timeout=${timeout}`
+      )
+      .then((res) => res.data)
+  }
+
+  promqlQueryRange(query: string, start: number, end: number, step: string) {
+    return client
+      .getAxiosInstance()
+      .get(
+        `/slow_query/vm_query_range?query=${query}&start=${start}&end=${end}&step=${step}`
+      )
+      .then((res) => res.data)
+  }
```

Comment:
I'm not sure, whether can we define these APIs in the tidb-dashboard backend so we can use the generated API methods.

Reply:
It looks like a special api from ngm.